resources.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+package gossamer
```

Comment:
Having this file at the root of the project isn't that great-great, but it's necessary to have it as a parent of the chains subdirectories. We could have it in the ./chain directory but that would mean changing all default paths. Since we need to change those paths (if we have that resource file in a subdir), maybe we can move those chain files in `./internal/chain/` and update the default paths? That way:

- Embedding is modular (if we later embed more stuff, we can pick what to embed depending on subpackage imports)
- `DefaultConfigTomlFiles` is not importable by other Go projects

Reply:
I personally like the changes with `internal/chain`, but let's keep it unresolved in case someone doesn't / see something wrong with it 🤔